packages/components/src/Gallery/Gallery.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import styles from "./Gallery.css";
+import { GalleryProps } from "./GalleryTypes";
+import { LightBox } from "../LightBox";
+import { FormatFile } from "../FormatFile";
+import { Button } from "../Button";
+
+export function Gallery({ files, size = "default", max }: GalleryProps) {
+  const images = files
+    .filter(file => file.type.startsWith("image/"))
+    .map(file => {
+      return { title: file.name, url: file.src };
+    });
+
+  const [lightboxOpen, setLightboxOpen] = useState(false);
+  const [lightboxIndex, setLightboxIndex] = useState(0);
+  const [displayPastMax, setDisplayPastMax] = useState(false);
+
+  return (
+    <>
+      <div className={size === "large" ? styles.galleryLarge : styles.gallery}>
+        {files.map((file, index) => {
+          if (max && !displayPastMax && index + 1 > max) {
+            return <></>;
+          }
+          return (
+            <div key={file.key}>
+              <FormatFile
+                file={{ ...file, src: () => Promise.resolve(file.src) }}
+                display="compact"
+                displaySize={size}
+                onClick={() => {
+                  handleThumbnailClicked(index);
+                }}
+                onDelete={file.onDelete}
+              />
+            </div>
+          );
+        })}
+        {max && files.length > max && !displayPastMax && (
+          <Button
```

Comment:
If we wrap a div around this Button with a width of {thumbnailSize (56 / 168)} and `display: flex`, and set the Button property to `fullWidth`, we'll get the square proportion that matches the design:

![image](https://user-images.githubusercontent.com/39704901/172462073-b20c33f7-9f97-4ae7-8c65-5ec38f5c57b3.png)
![image](https://user-images.githubusercontent.com/39704901/172462663-7094b583-13dd-4386-8c3e-1373c5d59f3f.png)

Reply:
We could also set the button to be `large` when the Gallery is large to make the "+N" label more prominent